docs/content/NewButton.mdx

```diff
@@ -0,0 +1,169 @@
+
+```jsx live
+<NewButton sx={{width: '100%'}}>Block</NewButton>
+```
+
+## API reference
+
+Native `<button>` HTML attributes are forwarded to the underlying React `button` component and are not listed below.
+
+### Button
+
+| Prop name    | Type                      |  Default  | Description                                                                                                                               |
+| :----------- | :------------------------ | :-------: | :---------------------------------------------------------------------------------------------------------------------------------------- |
+| children\*   | React.ReactNode           |           | Button description along with other components like Counter could be used here.                                                           |
+| variant      | String                    | 'default' | Changes the look and feel of the button which is different for each variant. One of `default`,`primary`, `outline`, `invisible`, `danger` |
+| size         | String                    | 'medium'  | a value of `small`, `medium`, or `large` results in smaller or larger Button text size                                                    |
+| leadingIcon  | Component                 |           | provide an octicon. It will be placed before the button text                                                                              |
+| trailingIcon | Component                 |           | provide an octicon. It will be places after the button text                                                                               |
+| icon         | Component                 |           | provide an octicon. This will ignore the text on the button and show an icon-only button.                                                 |
+| sx           | `BetterSystemStyleObject` |           | accepts all [styled system props](https://styled-system.com/table/).                                                                      |
+
+### Button.Counter
+
+| Prop name  | Type                      | Default | Description                                                          |
+| :--------- | :------------------------ | :-----: | :------------------------------------------------------------------- |
+| children\* | Number                    |         | The child for this has to be a number                                |
+| sx         | `BetterSystemStyleObject` |         | accepts all [styled system props](https://styled-system.com/table/). |
+
+## Component Status
+
+<ComponentChecklist
+  items={{
+    propsDocumented: true,
+    noUnnecessaryDeps: true,
+    adaptsToThemes: true,
+    adaptsToScreenSizes: true,
+    fullTestCoverage: true,
+    usedInProduction: null,
+    usageExamplesDocumented: true,
+    designReviewed: null,
+    a11yReviewed: null,
+    stableApi: false,
+    addressedApiFeedback: false,
+    hasDesignGuidelines: null,
+    hasFigmaComponent: null
```

Comment:
`null` here means "not applicable" but I think all of these "null" cases are things we want to eventually be "true"
```suggestion
    usedInProduction: false,
    usageExamplesDocumented: true,
    designReviewed: false,
    a11yReviewed: false,
    stableApi: false,
    addressedApiFeedback: false,
    hasDesignGuidelines: false,
    hasFigmaComponent: false
```